### PR TITLE
adding the language selector. UI needs to be cleaned up though

### PR DESCRIPTION
**Pull request by gibtang**

### CODE REVIEW

**Comment by samueljklee** [comment]
Clicking `Malay` twice will produce this URL: `https://deploy-preview-22--corona-tracker.netlify.com/ms/ms` which doesn't exist.

File: components/Nav.vue [hunk 1]
```diff
@@ -12,6 +12,7 @@
           >
             {{ $t(link.display) }}
           </nuxt-link>
+          <LanguageSelector />
```
Comment (samueljklee) [inline components/Nav.vue]: Mobile shows repeated language selector. need logic to check if not `showMobileLinks`?
image

File: components/LanguageSelector.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="relative px-1 py-2 cursor-pointer">
+    <div class="flex px-2 py-2 bg-gray-200" @click="showOptions" v-on-clickaway="closeOptions">
```
Comment (samueljklee) [inline components/LanguageSelector.vue]: add `rounded` for similar button "theme"
```suggestion
    <div class="flex px-2 py-2 bg-gray-200 rounded" @click="showOptions" v-on-clickaway="closeOptions">
```

Comment (xming13) [reply components/LanguageSelector.vue]: @samueljklee  Added. thanks!

**Pull request merged**